pages/linux/mkswap.md

```diff
@@ -0,0 +1,11 @@
+# mkswap
+
+> Sets up a Linux swap area on a device or in a file.
+
+- Setup a disk partition as swap area:
+
+`sudo mkswap {{partition}}`
+
+- Setup a file and use that file as swap space:
```

Comment:
What does it mean to "Setup a file" ? Does it make sense to just say `Use a given file as swap area:` ?

Reply:
done!